File: Regular expressions in Python/step02/2.06/2.6.04.py
'''

Напишите регулярное выражение, которое найдёт все цифры из шестнадцатеричной системы счисления.

Что нужно найти:

Цифры из шестнадцатеричной системы счисления: 0123456789ABCDEF

Sample Input 1:

<=>@ABCDEFGHIJKLMNOPQRSTUVWXYZ[]^_`abcdefghijklmnopqrstuvwxyz{|}~абвгдеёжзийклмнопрстуфхцчшщъыьэюяАБВГДЕЁЖЗИЙКЛМНОПРСТУФХЦЧШЩЪЫЬЭЮЯ !"#$%&\'()*+,-./0123456789:;

Sample Output 1:

A B C D E F 0 1 2 3 4 5 6 7 8 9

Sample Input 2:

#03B6FC

Sample Output 2:

0 3 B 6 F C

Sample Input 3:

٠١٢٣٤٥٦٧٨٩

Sample Output 3:

'''

regex = r'[0-9A-F]'

File: Diving into JavaScript: For Beginners/step02/2.06/2.6.04.js
/*

Запросите у пользователя число a, а затем число b. 
Проверьте является ли число а меньше числа b и присвойте результат проверки константе isLess. 
Выведите значение константы isLess.

Sample Input 1:

8
11

Sample Output 1:

true

Sample Input 2:

7
5

Sample Output 2:

false

Sample Input 3:

6
6

Sample Output 3:

false

*/

let a = Number(prompt());
let b = Number(prompt());

const isLess = a < b;
console.log(isLess);
